src/components/pages/rules/flyouts/ruleEditor/ruleEditor.js

```diff
@@ -394,17 +396,30 @@ export class RuleEditor extends LinkedComponent {
           </div>
         }
         <SummarySection>
-          {
-            isPending && <Indicator size="large" pattern="bar" />
-          }
-          <SummaryCount>{devicesAffected}</SummaryCount>
-          <SectionDesc>{t('rules.flyouts.ruleEditor.devicesAffected')}</SectionDesc>
+          <SectionHeader>{t('rules.flyouts.ruleEditor.summaryHeader')}</SectionHeader>
+          <SummaryBody>
+            <SummaryCount>{devicesAffected}</SummaryCount>
+            <SectionDesc>{t('rules.flyouts.ruleEditor.devicesAffected')}</SectionDesc>
+            {isPending && <Indicator />}
+            {
+              /*
+              TODO: Change interaction pattern.
+              - Make the flyout stay open to give the user visual confirmation of success.
+                    completedSuccessfully && <Svg className="summary-icon" path={svgs.apply} />
+              - Update the redux store in the background.
+              - Also, allow for additional changes to be made while the flyout is open.
+              */
+            }
+          </SummaryBody>
         </SummarySection>
+
         {
-          error && <AjaxError t={t} error={error} />
+          error &&
+          <AjaxError t={t} error={error} />
```

Comment:
leave this as oneline

Reply:
I was making these match the error display in devices... so now, I'll go change the devices ones too